lib/crypto.js

```diff
@@ -18,300 +18,67 @@ const getFipsCrypto = binding.getFipsCrypto;
 const setFipsCrypto = binding.setFipsCrypto;
 const timingSafeEqual = binding.timingSafeEqual;
 
-const Buffer = require('buffer').Buffer;
-const stream = require('stream');
-const util = require('util');
-const LazyTransform = require('internal/streams/lazy_transform');
-
-const DH_GENERATOR = 2;
-
 Object.defineProperty(exports, 'constants', {
   configurable: false,
   enumerable: true,
   value: constants
 });
 
+Object.defineProperty(exports, 'DEFAULT_ENCODING', {
```

Comment:
Yes, I thought I had already labeled it that way but looks like I missed it. Added!